code/modules/clothing/head/misc_special.dm

```diff
@@ -141,35 +146,14 @@
 	brightness_on = 2
 	light_overlay = "helmet_light"
 	w_class = ITEM_SIZE_NORMAL
+	style_coverage = COVERS_WHOLE_HEAD
 
-/*
- * Kitty ears
- */
-/obj/item/clothing/head/kitty
-	name = "kitty ears"
-	desc = "A pair of kitty ears. Meow!"
-	icon_state = "kitty"
-	body_parts_covered = 0
-	siemens_coefficient = 1.5
-	item_icons = list()
-
-/obj/item/clothing/head/kitty/equipped(mob/user, slot)
-	if(slot == slot_head)
-		update_icon(user)
-	..()
-
```

Comment:
Stealth removal?

Reply:
this would also runtime randomly whenever the guild artifact gets broken now.

Reply:
> this would also runtime randomly whenever the guild artifact gets broken now.

no it would not, as it is an entirely separate item.

Reply:
separate item where?

Reply:
/obj/item/clothing/head/collectable/kitty

Reply:
based!

Reply:
Cat ears removal is a good thing. Nobody likes felanids